app/models/setting.js

```diff
@@ -9,7 +9,7 @@ export default Model.extend(ValidationEngine, {
     title: attr('string'),
     description: attr('string'),
     logo: attr('string'),
-    cover: attr('string'),
+    cover_image: attr('string'),
```

Comment:
Is there a reason the API is inconsistent here? There's an odd mix of camelCase vs underscore_case...

```
ghost_head
ghost_foot
cover_image
```


```
defaultLang
activeTimezone
forceI18n
activeTheme
isPrivate
```